kratos/geometries/line_2d_2.h

```diff
@@ -792,6 +792,31 @@ class Line2D2 : public Geometry<TPointType>
     }
 
     /**
+     * It computes the normal of the geometry, if possible
+     * @return The normal of the geometry
+     */
+    virtual array_1d<double, 3> Normal() override
```

Comment:
here is a pointer to line 693 of geometry base class... no function called Normal(

[](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/geometries/geometry.h#L693)